### PR TITLE
[C-Api] define sub-plugin names

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix typo and add new function to get the nnfw type by subplugin name.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by again4you — APPROVED** [review]
LGTM!

File: api/capi/src/nnstreamer-capi-util.c [hunk 1]
```diff
@@ -982,27 +1001,13 @@ _ml_detect_framework (char **model, unsigned int num_models,
...
-    *nnfw = ML_NNFW_TYPE_VIVANTE;
-  } else if (g_ascii_strcasecmp (detected, "tensorflow") == 0) {
-    *nnfw = ML_NNFW_TYPE_TENSORFLOW;
-  } else if (g_ascii_strcasecmp (detected, "custom") == 0) {
-    *nnfw = ML_NNFW_TYPE_CUSTOM_FILTER;
-  } else {
-    ml_loge ("The given model has unknown or not supported extension.");
-    status = ML_ERROR_NOT_SUPPORTED;
-  }
+  *nnfw = ml_get_nnfw_type_by_subplugin_name (detected);
```
Comment (gichan-jang) [inline api/capi/src/nnstreamer-capi-util.c:1005]: Don't need to check whether `detected` NULL or not?

Comment (jaeyun-jung) [reply api/capi/src/nnstreamer-capi-util.c:1005]: Right! This will be checked in ml_get_nnfw_type_by_subplugin_name().

**Comment by jaeyun-jung** [comment]
2020-11-30 add simple testcase for nnfw type and name.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**